### PR TITLE
Update CONTRIBUTING.md

### PR DESCRIPTION
**Pull request by aditya1702**
I have tried to update the contributing file. If there is something missing or something incorrect please do let me know.

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -6,16 +6,23 @@ Need a push in the right direction?
...
     * Type ```vagrant ssh```
-* Make sure ```vagrant up``` works locally on your machine. If you are not sure, put the output in a [Gist](https://gist.github.com).
+    * Run the above commands the new folder you will create as your project folder. Make sure you do not have any            previous Vagrantfile created.
+ * Make sure ```vagrant up``` works locally on your machine. If you are not sure, put the output in a [Gist](https://gist.github.com).
 * Run the tests on the command line by ```python -m unittest discover```
 * Do they run correctly and pass ?. If not, let us know!
-* Read the [API Docs](https://github.com/fossasia/open-event/blob/master/API.md)
-* Do we have docs for each part of the system? If not, patches welcome!
\ No newline at end of file
+* Read the [API Docs](https://githbu.com/fossasia/open-event/blob/master/API.md)
```
Comment (creativepsyco) [inline CONTRIBUTING.md]: error

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -6,16 +6,23 @@ Need a push in the right direction?
...
+      also uswgi can be downloaded using the following command:
+                  pip install uwsgi
 * After installing the above dependencies change your working directory which you would like to use as your project directory.
 * Run the following command on the terminal
     * Type ```vagrant init```
     * Type ```vagrant up```
     * Type ```vagrant ssh```
-* Make sure ```vagrant up``` works locally on your machine. If you are not sure, put the output in a [Gist](https://gist.github.com).
+    * Run the above commands the new folder you will create as your project folder. Make sure you do not have any            previous Vagrantfile created.
+ * Make sure ```vagrant up``` works locally on your machine. If you are not sure, put the output in a [Gist](https://gist.github.com).
```
Comment (creativepsyco) [inline CONTRIBUTING.md]: one backtick is enough

**Comment by creativepsyco** [comment]
It would be better if you could look into some of the issues https://github.com/fossasia/open-event-orga-server/issues

**Comment by aditya1702** [comment]
Ok. I will look into the issues

Thank you

**Pull request closed without merging**